azure-devops/run-build.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         cmakeListsTxtPath: '$(Build.SourcesDirectory)/CMakeLists.txt'
         buildDirectory: $(Build.ArtifactStagingDirectory)/${{ parameters.targetPlatform }}
         useVcpkgToolchainFile: true
-        cmakeAppendedArgs: '-G Ninja -DENABLE_XUNIT_OUTPUT=TRUE'
+        cmakeAppendedArgs: '-G Ninja -DENABLE_XUNIT_OUTPUT=TRUE -DADDITIONAL_LIT_FLAGS=-j14'

```

Comment:
`14` is a mysterious number; can/should this be commented? The various timeout values are self-explanatory, but this relates to details of our infrastructure.

Reply:
I could use `NUMBER_OF_PROCESSORS` I guess.